### PR TITLE
thrift: update version to 0.13.0

### PR DESCRIPTION
**Pull request by ra1nb0w**
#### Description

update thrift to 0.13.0

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 10.15 19A602
Xcode 11.2 11B44

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by ra1nb0w** [comment]
@michaelld can you check if gnuradio needs revision bump (and if compile)? thanks

**Comment by michaelld** [comment]
@ra1nb0w sure will do

File: devel/thrift/Portfile [hunk 1]
```diff
@@ -122,6 +114,33 @@ variant erlang description "enable the Erlang library" {
...
+    depends_lib-append      port:rust
+}
+
+variant nodejs description "enable the NodeJS library" {
+    configure.args-delete   --with-nodejs=no
+    configure.args-append   --with-nodejs=yes
+    depends_lib-append      port:nodejs12
+}
+
+variant nodejs description "enable the Swift library" {
```
[the remark this answers is not in the document]
Comment (ra1nb0w) [reply devel/thrift/Portfile]: ops, sorry. done.
another thing that `port lift` needs to check.

[the remark this answers is not in the document]
Comment (michaelld) [reply devel/thrift/Portfile]: `port lint`? Agreed! and thx for promptly getting to this!

[the remark this answers is not in the document]
Comment (ra1nb0w) [reply devel/thrift/Portfile]: yes, `port lint` doesn't warn about a duplicated variant

**Comment by michaelld** [comment]
There are some issues with GR & this updated Thrift ... I'm debugging.

**Comment by michaelld** [comment]
Not many changes are required for building ... basically: `thrift::concurrency::PlatformThreadFactory` -> `thrift::concurrency::ThreadFactory` ... just a simple name change looks to work! Now have to work out the patch to handle both older and newer ... fun fun fun!

**Comment by michaelld** [comment]
OK I've got a tentative patch for GR37 & devel. I need to port it to next still. Should be able to work on it tomorrow.

**Comment by ra1nb0w** [comment]
ping

**Comment by michaelld** [comment]
Oh lordy! Sorry ... this one just got lost in my queue! Let me get GR fixed up first, then I'll deal with this. The PR itself LGTM.

**Comment by pmetzger** [comment]
@michaelld Any news?

**Comment by ra1nb0w** [comment]
thank you.

**Comment by michaelld** [comment]
YW! Sorry it took so long!

**Comment by ra1nb0w** [comment]
don't worry ;-)

**Pull request merged**